src/sentry/workflow_engine/handlers/detector/base.py

```diff
@@ -122,9 +106,6 @@ def __init__(self, detector: Detector):
 
     @abc.abstractmethod
     def evaluate(
-        self, data_packet: DataPacket[PacketT]
+        self, data_packet: DataPacket[DataPacketType]
     ) -> dict[DetectorGroupKey, DetectorEvaluationResult]:
         pass
-
-    def commit_state_updates(self):
-        pass
```

Comment:
is this removed because it's only for stateful detectors?

Reply:
💯